models/stats.go

```diff
@@ -4,10 +4,19 @@ import (
 	"database/sql/driver"
 	"encoding/json"
 	"errors"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"time"
 )
 
 // Regions (collections)
-var Regions = []string{"MDW", "FRA", "SIN", "NYC", "LAX", "LON", "PRG", "SAO", "ATL", "HND", "MAD", "MIA", "MOS2", "STO", "SYD", "SEA", "TOR", "SJO", "ASH", "HOU", "ATL2"}
+var (
+	Regions           []string
+	RegionsLastUpdate time.Time
+)
```

Comment:
```suggestion
	Regions           []string
	RegionsLastUpdate time.Time
	mu sync.RWMutex
)
```

I believe you need mutex, because these fields are use concurrently.